helpers/validators.php

```diff
@@ -28,4 +28,29 @@ function is_time_valid($time) {
 // Combines a date and time.
 function format_date_and_time($date, $time) {
     return $date . ' ' . $time;
+}
+
+// Makes sure the password is strong enough.
+// A password is at least 8 characters long.
+// A password must contain a letter
+// A password must contain a number
+// A password must contain a symbol.
```

Comment:
misschien kan dit beter special character zijn. is naar mijn mening iets duidelijker dan symbool.